src/views/Settings.vue

```diff
@@ -13,7 +13,10 @@
         <ion-label>{{ userProfile !== null ? userProfile.partyName : '' }}</ion-label>
         <ion-button slot="end" fill="outline" color="dark" @click="logout()">{{ $t("Logout") }}</ion-button>
       </ion-item>
-
+      <!-- OMS information -->
+      <ion-item>
+        <ion-label>OMS : {{instanceUrl}}</ion-label>
```

Comment:
Put instance url in ion note and put it in the "end" slot. Remove colon

Reply:
Done sir